### PR TITLE
ref(pageFilters): Make lock button more visible

### PR DESCRIPTION
**Pull request by vuluongj20**
**Before**
image

**After**
image

### CODE REVIEW

File: static/app/components/organizations/projectSelector/index.tsx [hunk 1]
```diff
@@ -65,10 +64,6 @@ type Props = {
    * Represents if a search is taking place
    */
   searching?: boolean;
-  /**
-   * Show the pin button in the dropdown's header actions
-   */
-  showPin?: boolean;
```
Comment (davidenwang) [inline static/app/components/organizations/projectSelector/index.tsx:71]: we might still need to maintain this prop until we completely remove the `selection-filters-v2` feature flag, or else for ST and self-hosted sentry users the pin/lock will show for the global header project selector which still exists until we completely remove the feature flag.

**Review by davidenwang — APPROVED** [review]
🌮 !!! looks good

File: static/app/components/organizations/projectSelector/index.tsx (hunk 1 above)
Comment (vuluongj20) [reply static/app/components/organizations/projectSelector/index.tsx]: true! added it back in https://github.com/getsentry/sentry/pull/34503/commits/aa9613937bf7097981e38926edd815ec5ef8f1d8

Comment (davidenwang) [reply static/app/components/organizations/projectSelector/index.tsx]: dang so quick 💯

**Review by dashed — APPROVED** [review]

**Pull request merged**